src/platform/packages/shared/kbn-scout/src/common/services/logger.ts

```diff
@@ -31,17 +30,27 @@ export class ScoutLogger extends ToolingLog {
   }
 }
 
-const loggerInstances = new Map<string, ScoutLogger>();
-
 /**
- * Singleton logger instance for specific worker to share across the Scout components
- * @param workerContext logger context, e.g. `scout-worker-1`; default is `scout`
- * @returns {ScoutLogger} logger instance
+ * Returns the resolved log level for Scout, prioritizing SCOUT_PW_LOG_LEVEL over LOG_LEVEL,
+ * and falling back to DEFAULT_LOG_LEVEL if neither is set or recognized.
  */
-export function getLogger(workerContext: string = 'scout'): ScoutLogger {
-  if (!loggerInstances.has(workerContext)) {
-    loggerInstances.set(workerContext, new ScoutLogger(workerContext));
+function resolveLogLevel(envValue: string | undefined): LogLevel | undefined {
+  if (typeof envValue === 'string' && envValue) {
+    let normalized = envValue.toLowerCase();
+    // Normalize common aliases for log levels
+    if (normalized === 'quiet') {
+      normalized = 'error';
+    }
+    const found = LOG_LEVEL_FLAGS.find(({ name }) => name === normalized);
+    if (found) return found.name as LogLevel;
   }
+  return undefined;
+}
 
-  return loggerInstances.get(workerContext)!;
+export function getScoutLogLevel(): LogLevel {
+  return (
+    resolveLogLevel(process.env.SCOUT_PW_LOG_LEVEL) ||
+    resolveLogLevel(process.env.LOG_LEVEL) ||
+    DEFAULT_LOG_LEVEL
+  );
```

Comment:
```suggestion
export function getScoutLogLevel(): LogLevel {
  return (
    resolveLogLevel(process.env.SCOUT_LOG_LEVEL) ||
    resolveLogLevel(process.env.LOG_LEVEL) ||
    DEFAULT_LOG_LEVEL
  );
```
IIUC, we don't want the log level for `ScoutLogger` and Playwright to be coupled:
- `SCOUT_LOG_LEVEL` overrides `DEFAULT_LOG_LEVEL`
- `SCOUT_PW_LOG_LEVEL` defaults to `SCOUT_LOG_LEVEL`